api/scpca_portal/management/commands/sync_original_files.py

```diff
@@ -22,11 +25,44 @@ def add_arguments(self, parser):
     def handle(self, *args, **kwargs):
         self.sync_original_files(**kwargs)
 
+    def generate_formatted_file_string(self, files: List[OriginalFile], file_type: str) -> str:
+        if not files:
+            return ""
+
+        formatted_file_str = f"{file_type}:\n"
+        formatted_file_str += "\n".join(f"- {str(f)}" for f in files) if files else "- None"
+        formatted_file_str += "\n"
+        return formatted_file_str
```

Comment:
Can we remove the `file_type` from this function so it just returns the multiline list of files or "None". Also, we shouldn't prefix the newline inside this method if we do change it and have the caller handle that.